x-pack/plugins/apm/server/routes/agent_keys/get_agent_keys.ts

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import { ApmPluginRequestHandlerContext } from '../typings';
+import { ApiKey } from '../../../../security/common/model';
+
+export async function getAgentKeys({
+  context,
+}: {
+  context: ApmPluginRequestHandlerContext;
+}) {
+  const body = {
+    size: 1000,
```

Comment:
We need a default size here

Reply:
We might want to make it configurable in the future